dbflow/src/main/java/com/xmartlabs/bigbang/dbflow/controller/DbFlowController.java

```diff
@@ -66,7 +66,7 @@
               });
           return entitiesToInsert;
         })
-        .subscribeOn(Schedulers.io());
+        .compose(applySingleIoSchedulers());
```

Comment:
Actually, we should keep the other ones, and we should remove the inheritance to the Controller class. If not, it forces us to inject the DbFlowController and it shouldn't be necessary